src/main/java/uk/gov/hmcts/ccd/domain/model/definition/CaseDetails.java

```diff
@@ -30,7 +30,8 @@
 import static uk.gov.hmcts.ccd.domain.model.definition.FieldTypeDefinition.CASE_PAYMENT_HISTORY_VIEWER;
```

Comment:
src/main/java/uk/gov/hmcts/ccd/domain/model/definition/CaseDetails.java  will be removed from the PR